server/config.py

```diff
@@ -38,7 +38,7 @@ def bool_env(val):
 # deprecated:
 CELERYBEAT_SCHEDULE = {
     "every-day": {
-        "task": "server.batch.charge_cards",
```

Comment:
`charge_cards` seems important, but I don't know if the system is handling recurring charges through some other system outside of itself (like Stripe recurring charges or something).